platform/view/services/db/driver/driver.go

```diff
@@ -59,6 +62,12 @@ type TransactionalVersionedPersistence interface {
 	NewWriteTransaction() (WriteTransaction, error)
 }
 
+type DBError error
+
+var (
+	UniqueKeyViolation DBError = errors.New("unique key violation")
```

Comment:
please, add a comment to this error. Please, state how this error should be used by a driver implementing the below interfaces